### PR TITLE
adds pa11y and precommit hooks

### PR DESCRIPTION
**Pull request by Danwhy**
ref #93
Does not include pa11y for CI

### CODE REVIEW

**Comment by shouston3** [comment]
Looks good 👍

**Review by nelsonic — APPROVED** [review]
😍

File: pa11y.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "hideElements": "iframe"
```
Comment (SimonLab) [inline pa11y.json]: From pa11y doc on hideElement (https://github.com/pa11y/pa11y#hideelements-string): "A CSS selector to hide elements from testing". What is the iframe on the application that we don't want to test?

**Review by SimonLab — APPROVED** [review]
LGTM, nice found about pa11y. For people wondering about the name: "Accessibility is often abbreviated as the numeronym a11y, where the number 11 refers to the number of letters omitted" see https://en.wikipedia.org/wiki/Computer_accessibility

File: pa11y.json (hunk 1 above)
Comment (shouston3) [inline pa11y.json]: When running the server in test/dev environment phoenix adds this iframe to the bottom of the html:
```html
<iframe src="/phoenix/live_reload/frame" style="display: none;"></iframe>
```

**Pull request merged**